libraries/postgres/src/Database.js

```diff
@@ -14,17 +15,49 @@ const EXTENSIONS = [
 // questionable decision can be overridden globally:
 pg.defaults.parseInputDatesAsUTC = true;
 
+defaultMonitorManager.register({
+  name: 'dbFunctionCall',
+  title: 'DB Method Call',
+  type: 'db-function-call',
+  level: 'info',
+  version: 1,
+  description: `This message is logged for each invocation of a Postgres stored function.`,
+  fields: {
+    name: 'The name of the DB function',
+  },
+});
+
+defaultMonitorManager.register({
+  name: 'dbPoolCounts',
+  title: 'DB Pool Counts',
+  type: 'db-pool-counts',
+  level: 'notice',
+  version: 1,
+  description: `
+    This message is logged every 5 minutes from each process that is using the database,
+    once for each pool.  Most services have both a "read" pool and a "write" pool.  The
+    fields come from the node-postgres package.
+  `,
+  fields: {
+    pool: 'The name of the pool within the process',
+    totalCount: 'The total number of connections',
+    idleCount: 'The number of connections (out of the total) which are not currently in use',
+    waitingCount: 'The number of operations waiting for an idle connection',
+  },
+});
+
```

Comment:
Should these be in a `monitor.js` file to be consistent with how the rest of services are written?

Reply:
For tc-lib-pulse and tc-lib-api, they're in the file where the logging calls are made, so I just followed that pattern.